kinetic/distribution.yaml

```diff
@@ -1617,6 +1617,22 @@ repositories:
       url: https://github.com/introlab/find-object.git
       version: kinetic-devel
     status: maintained
+  follow_waypoints:
+    doc:
+      type: git
+      url: https://github.com/danielsnider/follow_waypoints.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/danielsnider/follow_waypoints-release.git
+      version: 0.2.0-0
+    source:
+      test_pull_requests: true
```

Comment:
awesome thanks for confirming. The screenshot you shared is specific to the repository and not the organization, which makes me wonder if you added the user only for this repository or the entire organization. If you added it only to the repository, please make sure to follow the second bullet from the instructions "Alternatively you can setup the webhook manually for each repository" to create the webhook manually.
If you did it for the entire organization: all good :+1:  sorry for the noise.

In both cases: Welcome! and thanks for your contribution to ROS!